app/lib/actions/katello/repository/incremental_import.rb

```diff
@@ -0,0 +1,55 @@
+module Actions
+  module Katello
+    module Repository
+      class IncrementalImport < Actions::EntryAction
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        input_format do
+          param :id, Integer
+        end
+
+        # @param repo
+        # @param import_location location of files to import
+        def plan(repo, import_location)
+          action_subject(repo)
+
+          rpm_files = Dir.glob(File.join(import_location, "*rpm"))
+
+          json_files = Dir.glob(File.join(import_location, "*json"))
+          pulp_units = json_files.map { |json_file| JSON.parse(File.read(json_file)) }
+          errata = pulp_units.select { |pulp_unit| erratum? pulp_unit }
```

Comment:
Would there be other types of units such as package groups or puppet modules as part of an incremental?